src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.ts

```diff
@@ -208,6 +210,16 @@ export class JobResultTableComponent implements AfterViewInit, OnInit, OnDestroy
     return `https://www.metanetx.org/equa_info/${hp.replace('DM_', '')}`;
   }
 
+  showWarning(method: string, checked: boolean): void {
+    if (method === 'PathwayPredictor+DifferentialFVA' && !checked) {
```

Comment:
It's a little bit confusing that `checked` must be false to show the warning (you'd expect the warning to be shown when the row *is* checked), but I guess it's because `selection.isSelected(row)` checks the selection _before_ actually changing the state to checked.